addons/mail/static/src/models/thread/thread.js

```diff
@@ -571,6 +571,18 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * Performs the rpc to leave the rtc call of the channel.
+         */
+        async performRpcLeaveCall() {
+            this.update({ hasPendingRtcRequest: true });
+            await this.async(() => this.env.services.rpc({
+                route: '/mail/rtc/channel/leave_call',
+                params: { channel_id: this.id },
+            }, { shadow: true }));
+            this.update({ hasPendingRtcRequest: false });
```

Comment:
It is strange to see `hasPendingRtcRequest` written far from where it is used, and/or that there are calls to the method where we actually want to by-pass it, but those cases are still writing the field.